src/main/java/org/symphonyoss/symphony/messageml/elements/TextField.java

```diff
@@ -211,6 +212,28 @@ private void validateMinAndMaxLengths() throws InvalidInputException {
     if (minLength != null && maxLength != null && minLength > maxLength) {
       throw new InvalidInputException("The attribute \"minlength\" must be lower than the \"maxlength\" attribute");
     }
+    if (hasInitialValue()) {
+      TextNode initialValueElement = (TextNode) getChild(0);
+      String initialValue = initialValueElement.getText();
+      if (initialValue != null) {
+        if (isTextSmallerThanMinLength(minLength, initialValue) || isTextBiggerThanMaxLength(maxLength, initialValue)) {
+          throw new InvalidInputException(
+              "The initial value of a text field must respect the minlength and maxlength set on its element");
```

Comment:
we have both values, why don't we say "The length of the initial value must be between X and Y"?

Even if they don't pass the parameters, we have a fixed limit of 128 and the min of 1.